tests/rptest/tests/acls_test.py

```diff
@@ -166,76 +183,101 @@ def get_super_client(self):
         return RpkTool(self.redpanda,
                        username=username,
                        password=password,
                        sasl_mechanism=self.algorithm,
-                       tls_cert=cert)
-
-    # The old config style has use_sasl at the top level, which enables
-    # authorization. New config style has kafka_enable_authorization at the
-    # top-level, with authentication_method on the listener.
-    @cluster(num_nodes=3)
-    # plaintext conn + sasl for authn (global sasl config)
-    @parametrize(use_tls=False,
-                 use_sasl=True,
-                 enable_authz=None,
-                 authn_method=None)
-    # ssl/tls conn + sasl for authn (global sasl config)
-    @parametrize(use_tls=True,
-                 use_sasl=True,
-                 enable_authz=None,
-                 authn_method=None)
-    # ssl/tls conn + sasl for authn (listener sasl config)
-    @parametrize(use_tls=True,
-                 use_sasl=False,
-                 enable_authz=True,
-                 authn_method="sasl")
-    # ssl/tls conn + mtls for authn (listener mtls config)
-    @parametrize(use_tls=True,
-                 use_sasl=False,
-                 enable_authz=True,
-                 authn_method="mtls_identity")
-    # Disable authz
-    @parametrize(use_tls=True,
-                 use_sasl=True,
-                 enable_authz=False,
-                 authn_method=None,
-                 always_succeed=True)
-    # Disable authz
-    @parametrize(use_tls=True,
-                 use_sasl=True,
-                 enable_authz=False,
-                 authn_method="sasl",
-                 always_succeed=True)
-    # Disable authz
-    @parametrize(use_tls=True,
-                 use_sasl=True,
-                 enable_authz=False,
-                 authn_method="mtls_identity",
-                 always_succeed=True)
-    def test_describe_acls(self,
-                           use_tls,
-                           use_sasl,
-                           enable_authz,
-                           authn_method,
-                           always_succeed=False):
+                       tls_cert=self.base_user_cert)
+
+    '''
+    The old config style has use_sasl at the top level, which enables
+    authorization. New config style has kafka_enable_authorization at the
+    top-level, with authentication_method on the listener.
+
+    Brief param descriptions:
+        use_tls - Controls whether tls certs are used
+        use_sasl - Controls the value of enable_sasl RP config
+        enable_authz - Controls the value of kafka_enable_authorization RP config
+        authn_method - Controls the broker level authentication_method (e.g., mtls_identity)
+        client_auth - Controls the value of require_client_auth RP config
+    '''
+
+    @cluster(
+        num_nodes=3,
+        log_allow_list=[
+            re.compile(
+                "/var/lib/redpanda/redpanda.log: No such file or directory")
```

Comment:
Is this a workaround for a `grep` running when no log file was ever created?  That seems like something to be handled in the code that runs the grep rather than individual tests.

Reply:
Done